perf/README.md

```diff
@@ -34,3 +34,19 @@ TODO: issues on AMD until...zen3?
 ```sh
 perf report --no-children --sort=dso,symbol
 ```
+
+### Errors
+
+If you observe the error
+```
+Error:
+Invalid event (cycles:ppu) in per-thread mode, enable system wide with '-a'.
+```
+and happen to be on an AMD ZEN2 or older kernel, try dropping `:pp` from the
+`perf record` command, and switching from `--call-graph lbr` to
+`--call-graph fp`. This will generally require you to recompile your binary
+with frame pointers. LLVM can be profiled without any changes necessary IME.
```

Comment:
> LLVM can be profiled without any changes necessary IME.

This isn't quite precise.  We can get a list of hottest functions, but we don't get meaningful context.